rest/models/types.go

```diff
@@ -80,6 +80,55 @@ const (
 	Losers  Direction = "losers"
 )
 
+// AssetClass is an identifier for a group of similar financial instruments.
+type AssetClass string
+
+const (
+	AssetStocks  AssetClass = "stocks"
+	AssetOptions AssetClass = "options"
+	AssetCrypto  AssetClass = "crypto"
+	AssetFx      AssetClass = "fx"
+)
+
+// DataType is the type of data.
+type DataType string
+
+const (
+	DataTypeTrade DataType = "trade"
```

Comment:
can we delete `DataType` and `DividendType` from these consts?

Reply:
same issue as `AssetClass` where we already have a const for `Trade`

Reply:
For `DividendType` we already have a `LT` const too

Reply:
ahh, okay let's at least match the `AssetStocks` pattern then